.patches/strings/strings01.cairo.patch

```diff
@@ -0,0 +1,23 @@
+--- a/exercises/strings/strings01.cairo	2022-05-24 15:41:16.741793231 +0200
++++ b/exercises/strings/strings01.cairo	2022-05-24 15:41:30.545171320 +0200
+@@ -8,7 +8,9 @@
+ 
+ func decode_cipher1() -> (plaintext : felt):
```

Comment:
Can you provide a patch with 0 lines of context? This will help the solution be valid in the future, I just saw a single wording fix makes the test crash because of a uselessly too long patch file https://github.com/onlydustxyz/starklings/runs/6575254000?check_suite_focus=true